llvm/include/llvm/Demangle/ItaniumDemangle.h

```diff
@@ -445,6 +445,14 @@ class EnableIfAttr : public Node {
   }
 };
 
+#ifdef _MSC_VER
```

Comment:
Hm... is this a GitHub bug or this code will be added again?
It should be already in place - https://github.com/intel/llvm/pull/2021.

Reply:
This is to make compilation and testing for this PR pass. I will rebase and this will go away.